samples/javascript_nodejs/21.corebot-app-insights/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "corebot-app-insights",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Missing standard scripts:

    "scripts": {
        "start": "node ./index.js",
        "watch": "nodemon ./index.js",
        "lint": "eslint .",
        "test": "echo \"Error: no test specified\" && exit 1"

Reply:
Hi @scheyal, we just added the _start_, _watch_, and _lint_ scripts, and also fixed some ESLint errors.